parameters/tools/rviz_config.rviz

```diff
@@ -602,7 +602,7 @@ Visualization Manager:
       Size (Pixels): 3
       Size (m): 0.0399999991
       Style: Spheres
-      Topic: /hero/ed/navigation/cloud
+      Topic: /hero/depthimage_to_navscan_rgbd/navscan
```

Comment:
I am not sure the rgbd should be in this topic name. Topic should represent function, not implementation